packages/build/src/github-repo.ts

```diff
@@ -62,12 +62,13 @@ export class GithubRepo {
    * @returns {Promise<void>}
    * @memberof GithubRepo
    */
-  async createRelease(release: Release): Promise<void> {
+  async createDraftRelease(release: Release): Promise<void> {
```

Comment:
I guess we have to create a draft release since otherwise individuals' build variants assets won't get uploaded eh? schade, schade :/

Reply:
yep! .. maybe if doesn't work later we can upload the artifact from the download center to the release.